src/main/java/org/jenkinsci/plugins/pipeline/github/client/Review.java

```diff
@@ -0,0 +1,66 @@
+package org.jenkinsci.plugins.pipeline.github.client;
+
+import java.io.Serializable;
+import java.text.MessageFormat;
+
+import org.eclipse.egit.github.core.User;
+
+public class Review implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    public static final String STATE_APPROVED = "APPROVED";
+    public static final String STATE_PENDING = "PENDING";
+    public static final String STATE_CHANGES_REQUESTED = "CHANGES_REQUESTED";
+    public static final String STATE_DISMISSED = "DISMISSED";
+    public static final String STATE_COMMENTED  = "COMMENTED";
+
+    private String body;
```

Comment:
Can you add in `commitId`, here?

https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request

As someone pushes more changes to a PR, the new Reviews will be against different commit's, so it would be nice to be able to tell which one the review was against.

Reply:
https://github.com/jenkinsci/pipeline-github-plugin/pull/18/commits/7fa0d37d8b8edf0eed3027a65075f0e9b019c403